cms/settings/base.py

```diff
@@ -982,6 +982,7 @@
 SEARCH_API_MAX_PAGE_SIZE = int(os.getenv("SEARCH_API_MAX_PAGE_SIZE", "500"))
 
 # Auth
+SERVICE_AUTH_TOKEN = env.get("SERVICE_AUTH_TOKEN")
```

Comment:
question: should this be added to https://github.com/ONSdigital/dis-wagtail/blob/dca8fe7cd67e798b8cc807def36f77021bba54d7/docs/integrations/auth.md ?

Reply:
Done [`1504cfd` (#295)](https://github.com/ONSdigital/dis-wagtail/pull/295/commits/1504cfd783d2d2e0686c01faad308114bd136ed5)